obspy/fdsn/header.py

```diff
@@ -52,10 +53,13 @@ def __init__(self, value, server_info=None):
 
 FDSNWS = ("dataselect", "event", "station")
 
+if PY2:
+    platform_ = platform.platform().decode("ascii", "ignore")
+else:
+    platform_ = platform.platform().encode("UTF-8").decode("ascii", "ignore")
```

Comment:
Assuming I understand the problem correctly, then `"UTF-8"` should probably be replaced with 

```python
sys.getdefaultencoding()
```